test/data/parser_feature/raw_file.html

```diff
@@ -1,6 +0,0 @@
-<!doctype html>
```

Comment:
What is the behavior now for a file that doesn't contain a charset? I think you just removed this because of parser_console, but I wanted to double check.

(IIRC, we should default to utf-8, under the assumption that a server is giving us that in the HTTP header.)

Reply:
it defaults to `utf-8` now.